frontend/src/utils/deepCopy.ts

```diff
@@ -0,0 +1,3 @@
+export const deepCopy = <T>(original: T): T => {
+  return JSON.parse(JSON.stringify(original));
```

Comment:
WE NEEDED THIS HAHA 🚀

Reply:
HAHA yea, should probably change the implementation at some point though